### DISCUSSION TITLE
directus playground for issue purpose

### DISCUSSION THREAD

**Discussion by magomimmo** [directus/directus#3625] [Feature Requests]
Hi there,  while developing a new database intensive application by using docker-compose I set up non only the regular project, but even a kind to directus playground to design small experiments to practice and refine solutions to be later applied to much larger schema. I found this approach very useful also to discover directus's issues and make them easily reproducible.  When you create a small experiment with more M2M relationships between few collection (think about movie, actor and role, where the same actor could play more roles in the same movie) and you discover an issue, it would be much easier to report the issue by sharing that playground instead of writing long text, sometime even obscure, to report the issue. 

Is it just a silly idea to think about setting up a kind of directus playground for issue submission? 

That said I love directus! Keep going this way and thanks so much for your work.

**Comment by SalahAdDin** [comment]
It does not have any **Playground** yet, right?

**Reply by rijkvanzanten** [reply]
What type of playground are you referring to specifically? We currently don't have an easy way to spin up a new Directus copy to share with the core team, no, as that's a pretty difficult thing to share securely. As soon as you share the credentials to a demo instance on a public forum like GitHub, your instance will be destroyed with spam and other nefarious actions immediately (we've tried that in the past..)

**Reply by SalahAdDin** [reply]
Like the [playground offered by Strapi](https://docs.strapi.io/developer-docs/latest/plugins/graphql.html#usage), you can access it only in a development environment, and it is fair enough.

**Reply by rijkvanzanten** [reply]
You're talking about GraphiQL, that's something different from the feature request in this discussion

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
